chef/cookbooks/provisioner/recipes/dhcp_update.rb

```diff
@@ -21,9 +21,9 @@
              'if option arch = 00:06 {
        filename = "discovery/ia32/efi/bootia32.efi";
      } else if option arch = 00:07 {
-       filename = "discovery/x86_64/efi/bootx64.efi";
+       filename = "discovery/x86_64/efi/boot/grub2/x86_64-efi/core.efi";
```

Comment:
This will be generated by this command:
```
      bash "Build UEFI GRUB2 network boot images at net directory" do
        code "grub2-mknetdir --net-directory=#{tftproot} \
                             --subdir=discovery/#{arch}/#{uefi_subdir}/boot/grub2"
```